resources/views/platform/components/game/related-games-table.blade.php

```diff
@@ -0,0 +1,187 @@
+@props([
+    'consoles' => [],
+    'games' => [],
+    'sortOrder' => 'title',
+    'filterOptions' => [],
+    'userProgress' => null,
+])
+
+<div>
+    @if (count($consoles) < 1)
+        <p>No related games.</p><br/>
+    @else
+        <x-game.related-games-meta-panel
+            :selectedSortOrder="$sortOrder"
+            :filterOptions="$filterOptions"
+        />
+
+        @if ($userProgress !== null)
+        <?php
+            $addButtonTooltip = __('user-game-list.play.add');
+            $removeButtonTooltip = __('user-game-list.play.remove');
+        ?>
+        <script>
+            function togglePlayListItem(id)
+            {
+                $.post('/request/user-game-list/toggle.php', {
+                    type: 'play',
+                    game: id
+                })
+                .done(function () {
+                    $("#add-to-list-" + id).toggle();
+                    $("#remove-from-list-" + id).toggle();
+                    if ($("#add-to-list-" + id).is(':visible')) {
+                        $("#play-list-button-" + id).prop('title', '{{ $addButtonTooltip }}');
+                    } else {
+                        $("#play-list-button-" + id).prop('title', '{{ $removeButtonTooltip }}');
+                    }
+                });
+            }
+        </script>
+        @endif
+
+        @foreach ($consoles as $console)
+            @if ($filterOptions['console'])
+                <h2 class="flex gap-x-2 items-center text-h3">
+                    <img src="{{ getSystemIconUrl($console->ID) }}" alt="Console icon" width="24" height="24">
+                    <span>{{ $console->Name }}</span>
+                </h2>
+            @endif
+
+            <div><table class='table-highlight mb-4'><tbody>
+
+            <tr>
+                <th style='width:28%'>Title</th>
+                <th style='width:12%; cursor: help' class='text-right'
+                    title='The number of achievements in the set'>Achievements</th>
+                <th style='width:10%; cursor: help' class='text-right'
+                    title='The number of points associated to achievements in the set'>Points</th>
+                <th style='width:10%; cursor: help' class='text-right'
+                    title='As estimate of rarity for achievements in the set'>RetroRatio</th>
```

Comment:
```suggestion
                    title='An estimate of rarity for achievements in the set'>RetroRatio</th>
```

There is a minor typo here. I'm wondering if we should also remark that the higher the number is, the rarer unlocks for the set are (not sure what a great wording for this might be).